src/adapters/interport/execution/src/sdk/utils/constants.ts

```diff
@@ -0,0 +1,15 @@
+export const USDC_VAULT_ADDRESS = '0x22E7c5d443400eef48F58acdEC67986373cf6c74';
+export const USDT_VAULT_ADDRESS = '0xd9eF111006F7477aC6E70dD2B782D8A901f6Ff5b';
+export const STABLECOIN_FARM_ADDRESS = '0xEF902012C4a227461BdefF8Fa26b596eA125C711';
+
+export const RPC_URL = 'https://rpc.zklink.io';
+export const BACKUP_RPC_URL = 'https://rpc.zklink.network';
+
+export const SUBGRAPH_URL = 'http://3.114.68.110:8000/subgraphs/name/interport-stablecoinfarm-tracking-zklink-nova';
```

Comment:
Set the `SUBGRAPH_URL` as an environment variable, referencing the **example** provided.

Reply:
Done